Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_041.RabbitMqCredentialsValidator.cs

```diff
@@ -18,12 +18,19 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
         {
             if (!groups.TryGetNonEmptyValue("id", out FlexMatch id) ||
                 !groups.TryGetNonEmptyValue("host", out FlexMatch host) ||
-                !groups.TryGetNonEmptyValue("secret", out FlexMatch secret) ||
-                !groups.TryGetNonEmptyValue("resource", out FlexMatch resource))
+                !groups.TryGetNonEmptyValue("secret", out FlexMatch secret))
```

Comment:
We shouldn't need this if statement check for id, host, and secret now that these are required in the regex. Can just be string id = groups["id"].value for all 3